tests/test_util.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
+from numpy.testing import assert_allclose
+from numpy.testing import assert_array_equal
+import math
+import pytest
+import sfs
+
+cart_sph_data = [
+    ((1, 1, 1), (np.pi/4, np.arccos(1/np.sqrt(3)), np.sqrt(3))),
+    ((-1, 1, 1), (np.arctan2(1, -1), np.arccos(1/np.sqrt(3)), np.sqrt(3))),
+    ((1, -1, 1), (-np.pi/4, np.arccos(1/np.sqrt(3)), np.sqrt(3))),
+    ((-1, -1, 1), (np.arctan2(-1, -1), np.arccos(1/np.sqrt(3)), np.sqrt(3))),
+    ((1, 1, -1), (np.pi/4, np.arccos(-1/np.sqrt(3)), np.sqrt(3))),
+    ((-1, 1, -1), (np.arctan2(1, -1), np.arccos(-1/np.sqrt(3)), np.sqrt(3))),
+    ((1, -1, -1), (-np.pi/4, np.arccos(-1/np.sqrt(3)), np.sqrt(3))),
+    ((-1, -1, -1), (np.arctan2(-1, -1), np.arccos(-1/np.sqrt(3)), np.sqrt(3))),
+]
+
+
+@pytest.mark.parametrize('coord, polar', cart_sph_data)
+def testcart2sph(coord, polar):
+    x, y, z = coord
+    a = sfs.util.cart2sph(x, y, z)
+    assert_allclose(a, polar)
+
+
+@pytest.mark.parametrize('coord, polar', cart_sph_data)
+def testsph2cart(coord, polar):
+    alpha, beta, r = polar
+    b = sfs.util.sph2cart(alpha, beta, r)
+    assert_allclose(b, coord)
+
+
+direction_vector_data = [
+    ((np.pi/4, np.pi/4), (0.5, 0.5, np.sqrt(2)/2)),
+    ((3*np.pi/4, 3*np.pi/4), (-1/2, 1/2, -np.sqrt(2)/2)),
+    ((3*np.pi/4, -3*np.pi/4), (1/2, -1/2, -np.sqrt(2)/2)),
+    ((np.pi/4, -np.pi/4), (-1/2, -1/2, np.sqrt(2)/2)),
+    ((-np.pi/4, np.pi/4), (1/2, -1/2, np.sqrt(2)/2)),
+    ((-3*np.pi/4, 3*np.pi/4), (-1/2, -1/2, -np.sqrt(2)/2)),
+    ((-3*np.pi/4, -3*np.pi/4), (1/2, 1/2, -np.sqrt(2)/2)),
+    ((-np.pi/4, -np.pi/4), (-1/2, 1/2, np.sqrt(2)/2)),
+]
+
+
+@pytest.mark.parametrize('input, vector', direction_vector_data)
+def testdirection_vector(input, vector):
+    alpha, beta = input
+    c = sfs.util.direction_vector(alpha, beta)
+    assert_allclose(c, vector)
+
+
+db_data = [
+    ((0), -math.inf),
```

Comment:
The parentheses in `(0)` have no effect. They are especially not creating a `tuple`, which it looks like you wanted to do?

See https://nbviewer.jupyter.org/github/mgeier/python-audio/blob/master/intro-python.ipynb#Tuples